rootfs/docker-entrypoint-initdb.d/001_setup_envdir.sh

```diff
@@ -3,16 +3,23 @@
 cd "$WALE_ENVDIR"
 
 # access-key-id and access-secret-key files are mounted in via kubernetes secrets
-AWS_ACCESS_KEY_ID=$(cat access-key-id)
-AWS_SECRET_ACCESS_KEY=$(cat access-secret-key)
+if [ "$AWS_ACCESS_KEY_ID" == "" ]; then
+  AWS_ACCESS_KEY_ID=$(cat access-key-id)
+fi
+if [ "$AWS_SECRET_ACCESS_KEY" == "" ]; then
+  AWS_SECRET_ACCESS_KEY=$(cat access-secret-key)
```

Comment:
No. These keys are used by [wal-e](https://github.com/wal-e/wal-e) and [boto3](https://boto3.readthedocs.org/en/latest/guide/configuration.html#environment-variables) to set up the connection details to S3/minio.